Dockerfile/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Ubuntu Trusty Docker file for base TileDB Dependencies
+
+# This Dockerfile copies files from the parent directory, requiring a build
+# context from the parent directory. From `./..`, execute the following to
+# build:
+# docker build -f Dockerfile/Dockerfile -t tiledb:dev .
+#
+# Use the build arg 'enable' to configure optional TileDB components, e.g.:
+#   docker build -f Dockerfile/Dockerfile --build-arg enable=s3,hdfs -t tiledb:dev
+
+# Ubuntu Trusty
+FROM ubuntu:18.04
+
+# Optional components to enable (defaults to empty).
+ARG enable
+
+# Setup home environment
+RUN useradd tiledb
+RUN mkdir /home/tiledb && chown tiledb /home/tiledb
+ENV HOME /home/tiledb
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    wget \
+    unzip \
+    git \
+    cmake \
+    python3 \
+    python3-dev \
+    python3-pip \
+    libssl-dev \
+    && apt-get clean \
+    && apt-get purge -y \
+    && rm -rf /var/lib/apt/lists*
+
+RUN cd /tmp \
+    && wget https://github.com/Kitware/CMake/releases/download/v3.17.2/cmake-3.17.2-Linux-x86_64.tar.gz \
+    && tar -xzf cmake-3.17.2-Linux-x86_64.tar.gz \
+    && cp -R cmake-3.17.2-Linux-x86_64/bin /usr/ \
+    && cp -R cmake-3.17.2-Linux-x86_64/doc /usr/ \
+    && cp -R cmake-3.17.2-Linux-x86_64/man /usr/ \
+    && cp -R cmake-3.17.2-Linux-x86_64/share /usr/
+
+# Copy TileDB source files from the build context to `/home/tiledb/TileDB``
+COPY CMakeLists.txt /home/tiledb/TileDB/CMakeLists.txt
+COPY bootstrap /home/tiledb/TileDB/bootstrap
+COPY cmake /home/tiledb/TileDB/cmake
+COPY examples /home/tiledb/TileDB/examples
+COPY external /home/tiledb/TileDB/external
+COPY scripts /home/tiledb/TileDB/scripts
+COPY test /home/tiledb/TileDB/test
+COPY tiledb /home/tiledb/TileDB/tiledb
+
+# Install TileDB
+RUN cd /home/tiledb/TileDB \
+    && mkdir build \
+    && cd build \
+    && ../bootstrap --enable-debug --enable-verbose --enable-azure --enable-s3 --enable-serialization --enable=${enable} \
+    && make -j2 \
+    && make -j2 examples \
+    && make install-tiledb \
+    && echo 'export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/home/tiledb/TileDB/dist/lib' >> /home/tiledb/.bashrc
+
+RUN chown -R tiledb: /home/tiledb
+
+ENV LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
```

Comment:
Mismatch here -- the one above is `SRC/dist/lib` (default), but I don't think we install anything in `/usr/local/lib` unless specified with `--prefix`.

Reply:
Also, `/usr/local/lib` is a default value you do not need to add.  You can just run `ldconfig` after `make install`.

Reply:
Removed. This, and all other changes, were a copy+paste so I don't have any justification for why this existed.